doc/tutorial/ui.md

```diff
@@ -0,0 +1,422 @@
+(tutorial-ui)=
+# Getting started with the UI
+
+This tutorial gives a quick introduction to using the LXD UI.
+It covers installing and initializing LXD, getting access to the UI, and carrying out some standard operations like creating, configuring, and interacting with instances, configuring storage, and using projects.
+
+After going through these steps, you will have a general idea of how to use LXD through its UI, and you can start exploring more advanced use cases!
+
+```{note}
+Ensure that you have 20 GiB free disk space before starting this tutorial.
+```
+
+% Include content from [first_steps.md](first_steps.md)
+```{include} first_steps.md
+    :start-after: <!-- Include start tutorial installation -->
+    :end-before: <!-- Include end tutorial installation -->
+```
+
+## Access the UI
+
+You access the LXD UI through your browser.
+See {ref}`access-ui` for more information.
+
+1. Expose LXD to the network by setting the {config:option}`server-core:core.https_address` server configuration option:
+
+       lxc config set core.https_address :8443
+
+% Include content from [../howto/access_ui.md](../howto/access_ui.md)
+```{include} ../howto/access_ui.md
+    :start-after: <!-- Include start access UI -->
+    :end-before: <!-- Include end access UI -->
+```
+
+## Create and start instances
+
+Let's start by launching a few instances.
+With *instance*, we mean either a container or a virtual machine.
+See {ref}`containers-and-vms` for information about the difference between the two instance types.
+
+1. Select {guilabel}`Instances` from the navigation.
+1. Click {guilabel}`Create instance` to launch a container called `first` using the Ubuntu 24.04 LTS image:
+
+   ```{figure} /images/tutorial/create_instance.png
+   :width: 100%
+   :alt: Create an Ubuntu 24.04 LTS container
+   ```
+
+   To select the base image, click {guilabel}`Browse images`.
+   Click {guilabel}`Select` next to the Ubuntu 24.04 LTS image.
+
+   ```{note}
+   The images that are displayed are hosted on pre-configured {ref}`remote-image-servers`.
+   You can filter which images are displayed.
+
+   You can also upload a custom ISO file to boot from.
+   See {ref}`instances-create-iso` for more information.
+   ```
+
+1. Click {guilabel}`Create and start` to launch the container.
+
+   ```{note}
+   Launching this container takes a few seconds, because the image must be downloaded and unpacked first.
+   ```
+
+1. Launch another container called `second`, using the same image.
```

Comment:
Is the reason for creating the `second` container to demonstrate the increase in speed since the image is now locally available? If so, I think creating the `third` container (even without starting) would demonstrate the same behaviour? Perhaps we can reduce the steps here by removing the creation of `second` container? This would of course require some adjustments for the rest of the document.

Reply:
Good catch - I removed the third one now. In the CLI tutorial, we create the third one by copying the second, but we don't have that in the UI.